### DISCUSSION TITLE
“Ignore all” in Collabora Online on Nextcloud AIO

### DISCUSSION THREAD

**Discussion by karlemilnikka** [nextcloud/all-in-one#1652] [Questions] [answered]
Has anyone got the “Ignore all” option to work in Collabora Online on Nextcloud AIO? I’ve already [reported the issue to Collabora Online](https://github.com/CollaboraOnline/online/issues/5401), but I’m wondering if it is working for anyone at all or if it is a limitation of Nextcloud AIO (like e.g., the lack of support for custom dictionaries).

**Comment by szaimen** [comment, marked as answer]
Hi, I suspect that this is a general bug with the collabora docker container and not an AIO-specific problem as we simply reuse the official image and follow the official docs when setting it up.

**Reply by karlemilnikka** [reply]
Thanks for the quick reply. Just out of curiosity: does it work for you?

**Reply by szaimen** [reply]
yes, ignore-all seems to work in a test-document... The document is iin english and the default dictionaries are used.

**Reply by szaimen** [reply]
However it only works until you close the document. When reopening you need to re-add the ignore-all rules again...
